tests/CodeFormatterTests.cs

```diff
@@ -156,5 +156,22 @@ public async Task NoFilesFormattedWhenNotInList()
             Assert.Equal(0, formatResult.FilesFormatted);
             Assert.Equal(4, formatResult.FileCount);
         }
+
+        [Fact]
+        public async Task OnlyLogFormattedFiles()
+        {
+            var logger = new TestLogger();
+            var path = Path.GetFullPath("tests/projects/for_code_formatter/unformatted_solution/unformatted_solution.sln", SolutionPath);
+
+            var files = new[] {"Program.cs"};
+            var formatResult = await CodeFormatter.FormatWorkspaceAsync(logger, path, isSolution: true, logAllWorkspaceWarnings: false, saveFormattedFiles: false, filesToFormat: files, cancellationToken: CancellationToken.None);
+            var log = logger.GetLog();
+
+            var pattern = string.Format(Resources.Formatted_code_file_0, @"Program\.cs");
```

Comment:
I you formatted with the string `(.*)` instead, you could match with the pattern to extract the formatted file names out of the log.

Reply:
I think I understand what you're saying, but I made a slight adjustment to the test to find the file name.